src/mixed-line-bar-chart/chart-container.tsx

```diff
@@ -209,7 +209,6 @@ export default function ChartContainer<T extends ChartDataTypes>({
    */
   const highlightedPointRef = useRef<SVGGElement>(null);
   const highlightedGroupRef = useRef<SVGRectElement>(null);
-  const [isPlotFocused, setPlotFocused] = useState(false);
```

Comment:
`setPlotFocused` was being called when focusing the application (not the plot), so in reality this should have been called `[isApplicationFocused, setApplicationFocused]`. But the chart plot component already has its internal `isApplicationFocused` state, so that this duplication was not really necessary.